### ISSUE TITLE
Is winapi abandoned?

### ISSUE THREAD

**Issue by bbqsrc** [retep998/winapi-rs#867]
Hasn't been any commits or merges for 4 months or so, and quite a few important pending pull requests.

**Comment by retep998** [comment]
No, but I do lack the time and willpower to maintain `winapi` by myself currently.

Are you offering to help with the burden of maintaining `winapi`?

**Comment by bbqsrc** [comment]
Mm, are there any maintainers other than you at the moment?

**Comment by atouchet** [comment]
Is there any interest in moving winapi to rust-lang-nursery as suggested in #268?

**Comment by jtmorrisbytes** [comment]
@retep998 I am interested in using this library to help people write cross platform applications in rust and would be interested in helping further the development of this library.

**Comment by ratijas** [comment]
As a student of "Windows services in C++" Innopolis University course, I became highly enthusiastic about winapi and "rewriting it in rust". I am experimenting with Windows Performance Counters infrastructure at the moment, and writing some safe high-level [wrappers ](https://gist.github.com/ratijas/1eace10c0adec654b874d72a8eb0e536). I would like to offer my services, but I'm not sure how exactly I could help.

**Comment by retep998** [comment]
One possibility is having the Windows working group take over maintenance of `winapi` https://github.com/rust-windows/wg/issues/2

**Comment by ratijas** [comment]
@retep998 I don't see any action there. Who does this wg comprises? Or is it just a placeholder until some proactive people appear?

**Comment by DataTriny** [comment]
Hi @retep998 ,
How about asking more widely for maintainers by posting to This week in Rust for instance? There might be people who would be able to help you but who are simply unaware of this issue.

To me it looks like winapi-rs is slowing the whole Rust story on Windows.
